yaml-tests/src/test/resources/table-functions.metrics.yaml

```diff
@@ -0,0 +1,134 @@
+        (5, 6.0, 'bar')) as A(B, C, W(X, Y, Z))
+    explain: EXPLODE array((@c10 AS B, @c12 AS C, (@c15 AS X, promote(@c17 AS DOUBLE)
+        AS Y, @c19 AS Z) AS W), (@c24 AS B, @c26 AS C, (@c29 AS X, @c31 AS Y, @c33
+        AS Z) AS W)) | MAP (_.B AS B, _.C AS C, _.W AS W)
+    task_count: 68
+    task_total_time_ms: 20
+    transform_count: 21
+    transform_time_ms: 1
+    transform_yield_count: 3
+    insert_time_ms: 0
+    insert_new_count: 4
+    insert_reused_count: 0
+-   query: EXPLAIN select A.B, C, W from values (1, 2.0, (3, 4, 'foo')), (10, 90.2,
+        (5, 6.0, 'bar')) as A(B, C, W(X, Y, Z))
+    explain: EXPLODE array((@c12 AS B, @c14 AS C, (@c17 AS X, promote(@c19 AS DOUBLE)
+        AS Y, @c21 AS Z) AS W), (@c26 AS B, @c28 AS C, (@c31 AS X, @c33 AS Y, @c35
+        AS Z) AS W)) | MAP (_.B AS B, _.C AS C, _.W AS W)
+    task_count: 68
+    task_total_time_ms: 2
+    transform_count: 21
+    transform_time_ms: 0
+    transform_yield_count: 3
+    insert_time_ms: 0
+    insert_new_count: 4
+    insert_reused_count: 0
+-   query: EXPLAIN select A.B, C as Q, W.X from values (1, 2.0, (3, 4, 'foo')), (10,
+        90.2, (5, 6.0, 'bar')) as A(B, C, W(X, Y, Z))
+    explain: EXPLODE array((@c16 AS B, @c18 AS C, (@c21 AS X, promote(@c23 AS DOUBLE)
+        AS Y, @c25 AS Z) AS W), (@c30 AS B, @c32 AS C, (@c35 AS X, @c37 AS Y, @c39
+        AS Z) AS W)) | MAP (_.B AS B, _.C AS Q, _.W.X AS X)
+    task_count: 68
+    task_total_time_ms: 2
+    transform_count: 21
+    transform_time_ms: 0
+    transform_yield_count: 3
+    insert_time_ms: 0
+    insert_new_count: 4
+    insert_reused_count: 0
+-   query: EXPLAIN select * from (select A.B, C as Q, W.X from values (1, 2.0, (3,
+        4, 'foo')), (10, 90.2, (5, 6.0, 'bar')) as A(B, C, W(X, Y, Z))) as u
+    explain: EXPLODE array((@c20 AS B, @c22 AS C, (@c25 AS X, promote(@c27 AS DOUBLE)
+        AS Y, @c29 AS Z) AS W), (@c34 AS B, @c36 AS C, (@c39 AS X, @c41 AS Y, @c43
+        AS Z) AS W)) | MAP (_.B AS B, _.C AS Q, _.W.X AS X)
+    task_count: 100
+    task_total_time_ms: 4
+    transform_count: 33
+    transform_time_ms: 0
+    transform_yield_count: 4
+    insert_time_ms: 0
+    insert_new_count: 5
+    insert_reused_count: 0
+-   query: EXPLAIN select * from (select A.B, C as Q, W.X from values (1, 2.0, (3,
+        4, 'foo')), (10, 90.2, (5, 6.0, 'bar')) as A(B, C, W(X, Y, Z))) as u where
+        b < 8
+    explain: EXPLODE array((@c20 AS B, @c22 AS C, (@c25 AS X, promote(@c27 AS DOUBLE)
+        AS Y, @c29 AS Z) AS W), (@c34 AS B, @c36 AS C, (@c39 AS X, @c41 AS Y, @c43
+        AS Z) AS W)) | MAP (_.B AS B, _.C AS Q, _.W.X AS X) | FILTER _.B LESS_THAN
+        @c68
+    task_count: 100
+    task_total_time_ms: 7
+    transform_count: 33
+    transform_time_ms: 2
+    transform_yield_count: 4
+    insert_time_ms: 0
+    insert_new_count: 6
+    insert_reused_count: 0
+-   query: EXPLAIN select * from range(0, 11, 5)
+    explain: TFunc range()
```

Comment:
The explain still shows the previous version without the arguments. Since we only check on the metrics here that won't break the test run. It would still be better to regenerate the metrics files to reflect that latest change.